src/Components/Modal/Modal.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { observer } from "mobx-react";
+import { Button, Container, Col, Row } from "react-bootstrap";
+import { ErrorToast } from "../ErrorToast/ErrorToast";
+import type { MediaPresenter } from "../MediaPresenter";
+import styles from "./Modal.module.css";
+import classnames from "classnames";
+type Props = {
+  mediaPresenter: MediaPresenter;
+};
+
+export const Modal: React.FC<Props> = observer(({ mediaPresenter }) => {
+  const [modalOpen, setModalOpen] = useState<boolean>(false);
+  const openModal = (): void => {
+    setModalOpen(true);
+  };
+  const closeModal = (): void => {
+    var modal: HTMLElement | null = document.getElementById("magicModal");
+    if (modal != null) {
```

Comment:
you don't need these checks, you can just call `setModalOpen` directly

Reply:
Yepp made the changes, is there a reason why these are not desirable?